File: C#/facturas/Conexion.cs
using System.Data.SqlClient;

public static class Conexion
{
    public static SqlConnection ObtenerConexion()
    {
        string cadenaConexion = "jdbc:oracle:thin:@localhost:1521:xe";
        SqlConnection conexion = new SqlConnection(cadenaConexion);
        return conexion;
    }
}

File: FEL/src/com/factura/util/conexion.java
package com.factura.util;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.SQLException;

/**
 *
 * @author lreye
 */
public class conexion {
    
        static {
        try {
            //Class.forName("oracle.jdbc.driver.OracleDriver");
            Class.forName("oracle.jdbc.OracleDriver");
        } catch (ClassNotFoundException e) {
            e.printStackTrace();
        }
    }
        
    private static final String JDBC_DRIVER = "oracle.jdbc.driver.OracleDriver";
    private static final String DB_URL = "jdbc:oracle:thin:@localhost:1521:xe";
    private static final String USER = "system";
    private static final String PASS = "12345";

    public static Connection getConnection() throws SQLException {
        return DriverManager.getConnection(DB_URL, USER, PASS);
    }
    
}
